### ISSUE TITLE
Tool tip material default behavior on mobile device

### ISSUE THREAD

**Issue by ethan-gforces** [angular/material2#13536]
## Bug:
After recently upgrading to Angular 6 we have noticed that the default behavior Angular material is no longer functioning as expected.

Works as expected on desktop. However, on mobile when doing a long key press the tool tip does not show, The only way we have managed to hack it is adding a click to toggle the tooltip (which is not really ideal as its not the default behaviour).

### Code to reproduce the issue:
https://github.com/ethan-gforces/material-tooltip

### Detailed Reproduction Steps: 
1. Compile code
2. Navigate to app on a mobile device
3. The first tooltip does not display on mobile but does on desktop
4. Second tooltip displays on both mobile and desktop but not using the default behaviour (long keypress)

### What is the expected behavior?
With a long keypress (1.5 secs) the first tooltip will display

### What is the current behavior?
With a long keypress the first tooltip does not display

### Versions of Angular, Material and browsers are affected?
- Angular **6.1.9**
- Angular material **6.4.7**
- HammerJS **2.0.8**
- Google chrome and Samsung internet on mobile device

**Comment by crisbeto** [comment]
The tooltip's behavior on mobile depends on the `longpress` gesture from Hammerjs. From what I can tell, you don't have Hammer included.

**Comment by ethan-gforces** [comment]
You are correct @crisbeto  I had not in the committed version. However I had in my local copy (now committed) I have now also embellished the details on the issue and included screenshots on the tooltip docs example to show the same thing. Only on a long page the tooltip is quite often not even visible.

**Comment by RezaRahmati** [comment]
newbee question: how can I find version a bug is fixed in github?
